test/tall/regression/0500/0500.unit

```diff
@@ -28,19 +28,18 @@
       env.initialize(
         names[i],
         getter: (TopLevelBinding binding, ExprCont ek, ExprCont k) {
-          binding.getter =
-              (TopLevelBinding _, ExprCont ek0, ExprCont k0) => ek0(
-                "Reading static variable '${binding.name}' during its initialization",
-              );
+          binding
+              .getter = (TopLevelBinding _, ExprCont ek0, ExprCont k0) => ek0(
```

Comment:
This one looks a little strange to me. Is there anything we can do to nudge it towards the formatting on line 36?

Reply:
This one tends to get funny because the string literals inside the function literal doesn't fit in the page width. So the formatter goes a little wonky trying to cram as much in as it can. I think it's mostly a red herring.